include/range/v3/action/insert.hpp

```diff
@@ -76,23 +76,76 @@ namespace ranges
                 return unwrap_reference(cont).insert(p, n, std::forward<T>(t));
             }
 
+            namespace impl {
```

Comment:
This namespace is embedded inside the `adl_insert_detail` namespace, which is already wrapped in `\cond`/`\endcond`.